Frameworks/SPMySQLFramework/Source/SPMySQLConnection Categories/Querying & Preparation.m

```diff
@@ -693,6 +693,10 @@ - (void)_updateLastErrorMessage:(NSString *)theErrorMessage
 		theErrorMessage = [self _stringForCString:mysql_error(mySQLConnection)];
 	}
 
+    // VERY IMPORTANT to set to nil here
+    // DO NOT remove
+    queryErrorMessage = nil;
```

Comment:
ohh, interesting... I wonder if it works also if you put it into `else` below? It doesn't make sense to set nil and then set it, but makes sense to nil-it in else condition

Reply:
yes it works